tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -531,6 +544,33 @@ module.exports = {
           disabled = result.value
         })
       return disabled
+    },
+    /**
+     * checks if the required label is present in the expiration date field
+     * @return Boolean
+     */
+    isRequiredLabelPresent: async function () {
+      await this
+        .waitForElementVisible('@requiredLabelInCollaboratorsExpirationDate')
+      return true
+    },
```

Comment:
does the function returns false if `required label` is not present? because ff the element fails to be present and visible in the specified amount of time, the test fails.

Reply:
fixed